src/components/playback/skipsegment.ts

```diff
@@ -9,49 +9,54 @@ import './skipbutton.scss';
 import dom from 'scripts/dom';
 import globalize from 'lib/globalize';
 import * as userSettings from 'scripts/settings/userSettings';
+import focusManager from 'components/focusManager';
+import layoutManager from 'components/layoutManager';
 
 interface ShowOptions {
     animate?: boolean;
     keep?: boolean;
+    focus?: boolean;
+}
+
+function onHideComplete(this: HTMLButtonElement) {
+    if (this) {
+        this.classList.add('hide');
+    }
 }
 
 class SkipSegment extends PlaybackSubscriber {
-    private skipElement: HTMLButtonElement | undefined;
+    private skipElement: HTMLButtonElement | null;
     private currentSegment: MediaSegmentDto | null | undefined;
     private hideTimeout: ReturnType<typeof setTimeout> | null | undefined;
 
     constructor(playbackManager: PlaybackManager) {
         super(playbackManager);
 
+        this.skipElement = null;
         this.onOsdChanged = this.onOsdChanged.bind(this);
     }
 
-    onHideComplete() {
-        if (this.skipElement) {
-            this.skipElement.classList.add('hide');
-        }
-    }
-
     createSkipElement() {
         if (!this.skipElement && this.currentSegment) {
-            const elem = document.createElement('button');
-            elem.classList.add('skip-button');
-            elem.classList.add('hide');
-            elem.classList.add('skip-button-hidden');
-
-            elem.addEventListener('click', () => {
-                const time = this.playbackManager.currentTime() * TICKS_PER_MILLISECOND;
-                if (this.currentSegment?.EndTicks) {
-                    if (time < this.currentSegment.EndTicks - TICKS_PER_SECOND) {
-                        this.playbackManager.seek(this.currentSegment.EndTicks);
-                    } else {
-                        this.hideSkipButton();
+            let buttonHtml = '';
+
+            buttonHtml += '<button is="emby-button" class="skip-button hide skip-button-hidden"></button>';
```

Comment:
You can use `createElement` approach and add `elem.setAttribute('is', 'emby-button');`:
https://github.com/jellyfin/jellyfin-web/blob/9d7532ceabb47bca34e535f96cfc68efad0c52c1/src/controllers/session/login/index.js#L301